scripts/vint.py

```diff
@@ -0,0 +1,119 @@
+# Script to convert Vint's Linter warnings to valid GitHub warning or error
+#
+# Copyright (C) 2021    Lucas Ritzdorf, Luca Zeuch
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+import json
+import subprocess
+import sys
+
+class Vint:
+    config = dict()
```

Comment:
Don't use initializers for mutable class attributes; the changes carry on to all instances which is undesirable:

```py
>>> class X:
...     y = []
...
>>> inst1 = X()
>>> inst1.append([])
>>> inst1.y.append(1)
>>> print(inst1.y)
[1]
>>> inst2 = X()
>>> print(inst2.y)
[1]
>>>
```